repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/intrafilterLogging/RequestLog.java

```diff
@@ -41,33 +44,58 @@
     String requestBody;
     HashMap<String, String> headers;
 
+    /**
+     * Constructor populates all of the fields necessary for logging.
+     * @param mutableHttpServletRequest {@link MutableHttpServletRequest}
+     * @param filterContext {@link FilterContext}
+     * @throws IOException if there's an issue converting the response body to a string
+     */
     public RequestLog(MutableHttpServletRequest mutableHttpServletRequest,
                       FilterContext filterContext) throws IOException {
 
         preamble = "Intrafilter Request Log";
         timestamp = new DateTime().toString();
-        currentFilter = filterContext.getFilterConfig().getId() + "-" + filterContext.getFilterConfig().getName();
+        currentFilter = getFilterDescription(filterContext.getFilterConfig());
```

Comment:
you shouldn't call instance methods in a constructor behavior can be erratic. Either make it a static method or better yet just do a simple ternary.